### DISCUSSION TITLE
Can't login to my postal dashboard

### DISCUSSION THREAD

**Discussion by jossybenson** [postalserver/postal#1642] [Help with using Postal] [answered]
After successful installation of my postal everything looks good, but I was not been able to login using both my server IP and domain name, I have added A record check everything and of course everything looks good, please I need help.

**Comment by willpower232** [comment, marked as answer]
When you say unable to login, can you see the form but don't know the details?

**Reply by jossybenson** [reply]
No login page at all! I try visiting it using my server IP nothing work, I still try using my domain name: joebenzmail.com not working I try mail.joebenzmail.com and postal.joebenzmail.com nothing work bro!

**Reply by willpower232** [reply]
It looks like you have a problem with nginx or caddy or whatever you're using.

![image](https://user-images.githubusercontent.com/1619102/137359690-94449d4e-717f-4340-9de0-457a39addb5e.png)

**Reply by jossybenson** [reply]
Oh ok so what will I do now?

**Reply by jossybenson** [reply]
Of course am using Caddy

**Reply by willpower232** [reply]
You have a problem with caddy so you should have a look in your logs or maybe it needs a restart. You could also share your caddy configuration, I don't know enough about it to know what your problem is off the top of my head.

**Reply by jossybenson** [reply]
Ok thank you so much I will send the caddy configuration with you later! I really appreciate your help thank you God bless you.

**Comment by jossybenson** [comment]
Please I need your help please

**Reply by LZL0** [reply]
I have the exact same issue.
Caddy fails to start and it just keeps restarting.

![image](https://user-images.githubusercontent.com/12474488/137410771-0ebd2bad-9c76-4b8e-8eb5-0b9e734890ee.png)

**Reply by LZL0** [reply]
Clean Ubuntu 20.04 LTS install. I only have this and an NGINX server on port 80.

**Reply by LZL0** [reply]
It starts just fine if I stop NGINX.

![image](https://user-images.githubusercontent.com/12474488/137411067-f21010e5-d44d-430a-bf57-69085e6c697a.png)

Any suggestions?

**Reply by pjv** [reply]
If nginx is bound to port 80, you can't then also bind caddy to port 80. You can only bind one process to a given port.

**Reply by jossybenson** [reply]
I have found the solution, first downgrade to ubuntu 18.04 then use putty to run the installation again and everything will work fine, right now am sending emails with no problem at all. If any need the command I use, I can post it here for free use.
